welcome/index.adoc

```diff
@@ -229,7 +228,7 @@ ifdef::openshift-dedicated[]
 
 - *Create container images*: A container image is the most basic building block in {product-title} (and Kubernetes) applications. By defining image streams, you can gather multiple versions of an image in one place as you continue its development. With S2I containers, you can insert your source code into a base container that is set up to run code of a particular type (such as Ruby, Node.js, or Python).
 
-- *Create deployments*:  Use `Deployment` and `DeploymentConfig` objects to exert fine-grained management over applications. Use the Workloads page or `oc` CLI to manage deployments. Learn rolling, recreate, and custom deployment strategies.
+- *Create deployments*:  Use `Deployment` objects to exert fine-grained management over applications. 
```

Comment:
@bergerhoffer i think we could still keep the line `Use the Workloads page or `oc` CLI to manage deployments. Learn rolling, recreate, and custom deployment strategies` right as it talks about deployments itself.